src/schema/v2/me/__tests__/myCollectionCreateArtworkMutation.test.ts

```diff
@@ -285,43 +285,6 @@ describe("myCollectionCreateArtworkMutation", () => {
 
       expect(message).toEqual(serverError)
     })
-
-    it("creates additional images in sequence to avoid a gravity race condition", async () => {
```

Comment:
This test's presence make me wonder if we need the optimisation above on using `Promise.all`. It looks like it was an intentional decision from David to go with it. I would suggest reverting this unless we can see a significant increase in performance with no side effects

Reply:
we thought about the same thing at the same time! thanks @olerichter00

Reply:
feel free to merge afterwards